commands/Char.js

```diff
@@ -0,0 +1,63 @@
+import { MessageEmbed } from "@yuuto-project/discord.js";
+import Command from "./base/Command.js";
+// Adapted from: https://github.com/dunste123/hirobot/blob/master/valentines.json
+import chars from "../assets/chars.json";
+
+export default class Char extends Command {
+  constructor() {
+    super({
+      name: "char",
+      category: "info",
+      aliases: ["character", "characters"],
+      description: "Show information about a specific character.",
+      usage: "Run `char <character>` to get details about a character."
+    });
+  }
+
+  run(client, message, args) {
+    const { channel } = message;
+
+    if (args.length === 0) {
```

Comment:
No need for a strict check, falsy is enough here
```suggestion
    if (!args.length) {
```